### DISCUSSION TITLE
Adding Script with Functions

### DISCUSSION THREAD

**Discussion by Yoofi-A-Appiah** [facebook/docusaurus#7834] [Q&A] [answered]
I want to add a script that includes a function to display ads. Is this possible?

**Comment by slorber** [comment, marked as answer]
You can swizzle React components and load display ad scripts in `useEffect`

I can't tell how easy it would be, but there are definitively Docusaurus sites running ads.

Redux-saga displays Carbon Ads in the docs sidebar for example: https://redux-saga.js.org/docs/introduction/GettingStarted

(open-source, you can inspect their site's code to check how it's integrated)
